src/JS.php

```diff
@@ -211,9 +211,7 @@ protected function extractRegex()
         };
 
         // it's a regex if we can find an opening and (not escaped) closing /,
-        // include \n because it may be there for a reason
-        // (https://github.com/matthiasmullie/minify/issues/56)
-        $pattern = '(\/.*?(?<!\\\\)(\\\\\\\\)*+\/\n?)';
+        $pattern = '(\/.*?(?<!\\\\)(\\\\\\\\)*+\/)';
```

Comment:
The `\n` must not be removed here. This change would case a side-effect of things like this being misinterpreted:
```js
a = 4 /
2;
```
With the new code, it would be minified to:
```js
a=4/;2;
```

The \n has to be included in here, because this is the place where regular expressions are extracted in a safe way. Anywhere else (e.g. preg_replaces in stripWhitespace), a `/` could be either division or regex, or part of the content of a string.